src/pages/StatsTracking.tsx

```diff
@@ -75,7 +82,9 @@ export function StatsTracking({ surveyUnits }: Readonly<StatsTrackingProps>) {
               <Select
                 options={sortOptions}
                 value={sortDirection}
-                onChange={handleSortChange}
+                onChange={(e: SelectChangeEvent<SetStateAction<SortDirection>>) =>
```

Comment:
It looks strange to have SetStateAction here. SetStateAction is not the type from the useState hook ?